mappings/net/minecraft/entity/mob/warden/WardenEntity.mapping

```diff
@@ -0,0 +1,32 @@
+CLASS net/minecraft/unmapped/C_fynaplbn net/minecraft/entity/mob/warden/WardenEntity
+	FIELD f_aklkpyqr lastHeartAnimationTicks I
+	FIELD f_azgepkis sniffingAnimation Lnet/minecraft/unmapped/C_kxntavoz;
+	FIELD f_ctsqwubn emergingAnimation Lnet/minecraft/unmapped/C_kxntavoz;
+	FIELD f_jaxcsvmw gameEventListener Lnet/minecraft/unmapped/C_wpaexsvv;
+	FIELD f_jctlypxu roaringAnimation Lnet/minecraft/unmapped/C_kxntavoz;
+	FIELD f_jhswzlgv tendrilAnimationTicks I
+	FIELD f_nmetugak diggingAnimation Lnet/minecraft/unmapped/C_kxntavoz;
+	FIELD f_pbinxyrw attackingAnimation Lnet/minecraft/unmapped/C_kxntavoz;
+	FIELD f_rtydirkh sonicBoomAnimation Lnet/minecraft/unmapped/C_kxntavoz;
+	FIELD f_vwjcyten ANGER_LEVEL Lnet/minecraft/unmapped/C_rinmcaxy;
+	FIELD f_wpbqvehb logger Lorg/slf4j/Logger;
+	FIELD f_xkfszxjq heartAnimationTicks I
+	FIELD f_zrbwwgvb angerManager Lnet/minecraft/unmapped/C_fjljvdfu;
+	METHOD m_aeewzlna diggingOrEmerging ()Z
```

Comment:
```suggestion
	METHOD m_aeewzlna isDiggingOrEmerging ()Z
```

Reply:
> Boolean field and variable names should always be adjective phrases or present tense verb phrases (`powered`, `canOpen`),
avoiding the `is` and `has` prefixes when possible (`colored`, not `isColored` or `hasColor`).

Reply:
I think is is better, we have been following the is format for a while